### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by wm901115nwpu** [yuanyuanli85/Fast_Human_Pose_Estimation_Pytorch#5]
Why do you define two loss function?
        total_loss   = loss_labeled + unkdloss_alpha * kdloss_unlabeled
        total_loss = kdloss_alpha * tsloss + (1 - kdloss_alpha)*gtloss

**Comment by wm901115nwpu** [comment]
the second should be deleted

**Comment by yuanyuanli85** [comment]
thank you for point it our.  yes, you are right.  there is a bug there. the 2nd loss should be deleted.  will fix it after some testing. thank you very much.

**Comment by yuanyuanli85** [comment]
fix has been commited by 38635cf
